scipy/optimize/_minimize.py

```diff
@@ -202,6 +204,20 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     and either the Hessian or a function that computes the product of
     the Hessian with a given vector.
 
+    Method :ref:`trust-region-exact <optimize.minimize-trustexact>`
+    uses nealy exact trust-region algorithm described in [13]_ for
```

Comment:
I would write something like "is a trust-region method for unconstrained minimization in which quadratic subproblems are solved almost exactly [13]_. This algorithm requires the gradient and the Hessian, which is *not* required to be positive definite". 

And that's probably it. Further details are to be found in the tutorial and the implementation docstring. I'm relying on how it is written for other methods in this section.